fedmsg_meta_fedora_infrastructure/taskotron.py

```diff
@@ -39,5 +46,45 @@ def link(self, msg, **config):
         if msg['topic'].endswith('taskotron.result.new'):
             return msg['msg']['result'].get('log_url', '')
 
+    def objects(self, msg, **config):
+        packages = self.packages(msg, **config)
+        return set([
+            '/'.join([
+                msg['msg']['task'].get('name', ''),
+                package,
+                msg['msg']['result'].get('outcome', '')
+            ])
+            for package in packages
+        ])
+    def packages(self, msg, **config):
+        type = msg['msg']['task']['type']
+        if type == 'koji_build':
+            nvr = msg['msg']['task']['item']
+            name, version, release = nvr.rsplit('-', 2)
```

Comment:
Acknowledged.  If resultsdb eventually starts including a list of packages along with its fedmsg payload, this code can be revised to drop the NVR parsing and use that list instead.